dedup-and-exclude.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env python3
+
+import sys
+
+def strip_whitespace(iterable):
+    for s in iterable:
+        s = s.strip()
+        if len(s) != 0:
+            yield s
+
+def stderr(s):
+    print(s, file=sys.stderr, flush=True)
+
+if len(sys.argv) == 3:
+    with open(sys.argv[1]) as f:
+        s = set(strip_whitespace(f))
+    max_lines = int(sys.argv[2])
+else:
+    program = sys.argv[0]
+    stderr(f"Usage: {program}: /path/to/excludes max-num-of-lines-to-output")
```

Comment:
(optional) inline this because it is only used once

Reply:
The function `stderr` is actually used in the loop as well.

Reply:
The function `stderr` is actually used in the loop as well.